### ISSUE TITLE
Update demo renderer for pixi.js v3

### ISSUE THREAD

**Issue by dlindenkreuz** [schteppe/p2.js#169]
The demo renderer in `demos/js/WebGLRenderer.js` is apparently still using pixi.js v1.4.2.

Updating the dependency to v3.0.6 gave me a lot of errors. After some temporary fixes, all object outlines / strokes were drawn with a constant offset (see screenshot). This only happens when the pixi.js WebGLRenderer is used, looks fine with the pixi.js CanvasRenderer.

I think this has something to do with camera and zoom transforms.

![bildschirmfoto 2015-06-26 um 17 59 57](https://cloud.githubusercontent.com/assets/509681/8381649/3b6ec560-1c2d-11e5-9089-a4dd215d78aa.png)

**Comment by dlindenkreuz** [comment]
The border offset thing is due to GoodBoyDigital/pixi.js#1620.

Still, the Renderer should be updated to work with pixi.js v3.0.6. Use the canvas renderer until resolved in pixi.js.

**Comment by schteppe** [comment]
Indeed. I made a quick and dirty hack to fix the outline offset in the older Pixi version.
Are your WebGLRenderer updates avalable somewhere?
Thanks for looking into this :+1:
